connectors/webhook/pom.xml

```diff
@@ -41,7 +41,27 @@
       <artifactId>guava</artifactId>
       <version>${version.guava}</version>
     </dependency>
-
+    <dependency>
+      <groupId>com.auth0</groupId>
+      <artifactId>java-jwt</artifactId>
+      <version>4.4.0</version>
```

Comment:
Shall we generalize versions one level above? The same as we have for `guava` for example?

Reply:
Good thinking, but the reason I put it here is because this is very specific for this connector. It is highly unlikely we will ever use this dependency in any other connector. WDYT?

Reply:
@markfarkas-camunda Agree, we can move it if its needed somewhere else.

Reply:
Good point

Reply:
I'm also good with defining the version in this pom. But I would still recommend to define the version in a property to follow the general code style